modules/scala/jupyter-api/src/main/scala/almond/display/Image.scala

```diff
@@ -108,14 +111,27 @@ final class Image private (
 
 object Image extends Display.Builder[Array[Byte], Image] {
 
+  def fromRenderedImage(image: RenderedImage, format: Format = JPG): Image = {
```

Comment:
Better add two overloads with no actual default values (one accepting both args, one accepting one arg calling the former), to make it easier to add extra fields in the future, while preserving binary compatibility.

Reply:
Without a default value we could also overload `Image.apply` here. Do you have a preference on this?

Reply:
Using to overloads now as suggested. Kept `fromRenderedImage` for now because it feels more consistent with `from` and `fromFile` in `Builder`.